pkg/reconciler/reconciler.go

```diff
@@ -632,14 +650,16 @@ func (r *Reconciler) doUpgrade(actionClient helmclient.ActionInterface, u *updat
 		}
 	}
 
-	rel, err := actionClient.Upgrade(obj.GetName(), obj.GetNamespace(), r.chrt, vals, opts...)
+	rel, err := actionClient.Upgrade(obj.GetName(), obj.GetNamespace(), r.chart, vals, opts...)
 	if err != nil {
 		u.UpdateStatus(
 			updater.EnsureCondition(conditions.Irreconcilable(corev1.ConditionTrue, conditions.ReasonReconcileError, err)),
 			updater.EnsureCondition(conditions.ReleaseFailed(corev1.ConditionTrue, conditions.ReasonUpgradeError, err)),
 		)
 		return nil, err
 	}
+	// Since the controller was not created WithEventRecorder will the event be recorded?
+	// Shows that not, so if the idea is let optional now then we need to do the impl for that and doc as well
```

Comment:
[We set a default event recorder](https://github.com/joelanford/helm-operator/blob/f37541622aa07ee8d397e0f6f90bd09dd9294717/pkg/reconciler/reconciler.go#L726-L728).